src/nodes/array/shift.js

```diff
@@ -0,0 +1,44 @@
+const dayjs = require('dayjs')
+const AbstractNode = require('../abstract')
+
+class ArrayShift extends AbstractNode {
+
+  static metadata() {
+    return {
+      name: 'Shift',
+      code: 'array/shift',
+      type: 'modifier',
```

Comment:
shift, pop, push, unshift are not modifiers.
they change values of inputs, so they should be callables and has execution flow input/output